src/rard/utils/convertors.py

```diff
@@ -11,28 +12,32 @@ class FragmentIsNotConvertible(Exception):
     pass
 
 
-def transfer_duplicates(source, destination, source_type):
-    """When converting between frags and anonfrags, the duplicate relationship needs to be transferred to the new (anon)fragment"""
-    if source_type == "fragment":
-        # for each duplicate create a new relationship to the anonfrag
-        for d in source.duplicates_list:
-            d.duplicate_afs.add(destination)
-            # if the relationship was defined from the duplicate in the list, remove the one being converted
-            if source in d.duplicate_frags.all():
-                d.duplicate_frags.remove(source)
-            else:
-                # otherwise delete it from the frag being converted
-                source.duplicate_frags.remove(d)
-    if source_type == "anonymous":
-        # for each duplicate create a new relationship to the frag
-        for d in source.duplicates_list:
-            d.duplicate_frags.add(destination)
-            # if the relationship was defined from the duplicate in the list, remove the one being converted
-            if source in d.duplicate_afs.all():
-                d.duplicate_afs.remove(source)
-            else:
-                # otherwise delete it from the anonfrag being converted
-                source.duplicate_afs.remove(d)
+def transfer_duplicates(source, destination):
```

Comment:
Updated in recent commit to ensure destination is added as duplicate of the source's duplicate as well as the reverse relationship being created.